rust-toolchain.toml

```diff
@@ -1,5 +1,5 @@
 [toolchain]
-channel = "stable"
+channel = "1.74"
```

Comment:
Is there a reason to change from `stable` to `1.74`?

Reply:
just to be consistent wrt risc0

Reply:
Ah, ya that makes sense 👍